### DISCUSSION TITLE
Hue Motion Sensors as Triggers

### DISCUSSION THREAD

**Discussion by ucfalmi** [MiguelRipoll23/homebridge-securitysystem#351] [Ideas]
Hey Miguel and contribs - please can you consider adding hue motion sensor alerts to the app?  Many people have 5-10 hue motion sensors in their homes which can be in Zones.  This would be an awesome way of adding an internal alarm system without needing any new hardware.  I believe the hue APIs and existing homebridge support should be easy to code features against and I'd love to test an alpha/beta.  Thanks!

**Comment by rvaneeuwijk** [comment]
I am already using Hue sensors as part of my setup. Basically I use HomeKit (incl Hue sensors), HOOBS to run SecuritySystem and Notify.Events and that's it: Hue sensors are part of the security system.

**Reply by ucfalmi** [reply]
Oh, great!  I got the Notify.Events plugin but I have no idea how to add channel or actions for Hue sensors (I'm a noob at this) - can you share some configs to get me started?  Thanks, I'd be really grateful - maybe Miguel could also host them as an example in the docs?

**Reply by rvaneeuwijk** [reply]
Basically, I have the following setup:

- In my Notify.Events account (web site), I define the channel and the recipients of messages to the channel.

- Hue Motion Alert is a message that I created for the channel in the config of the Notify.Events homebridge plugin.

- In Apple Home, I have an automation: if Hue motion sensor detects motion while Security System is in Away mode, then switch on Hue Motion Alert

**Reply by ucfalmi** [reply]
Hi and thanks for the reply.  It would be super-useful if you could share a screenshot of your homebridge action - this is the part I still don't understand.  I'm not even sure where to add "Hue Motion Alert is a message that I created for the channel in the config of the Notify.Events homebridge plugin."  Thank you!  @Admin could add some example configs for HomeKit motion sensors!  :-)

**Comment by MiguelRipoll23** [comment]
Hello

Aren't Hue sensors part of HomeKit? I just own Hue lights but I wasn't aware they couldn't be integrated easily.

**Reply by rvaneeuwijk** [reply]
Yes. I have loads of Hue lamps and Hue sensors and I have all of them integrated in HomeKit and all my automations set up in HomeKit (none in Hue).
